src/ocean/util/cipher/gcrypt/HMAC.d

```diff
@@ -32,12 +32,12 @@ module ocean.util.cipher.gcrypt.HMAC;
 
 import ocean.util.cipher.gcrypt.core.MessageDigestCore;
 import ocean.transition;
+import ocean.util.cipher.gcrypt.c.md;
 
 /******************************************************************************/
 
 public class HMAC: MessageDigestCore
 {
-    import ocean.util.cipher.gcrypt.c.md;
```

Comment:
This kind of changes is one of main reasons why I was reluctant to hurry with 2.071 update. Problem is that it doesn't only affect ocean - any downstream library that used to call `HMAC.some_imported_md_symbol` will break. I am going to ignore this commit for now and re-add it later after testing how it affects downstream projects.